doc/source/whatsnew/v0.25.2.rst

```diff
@@ -64,7 +64,7 @@ I/O
 
 - Fix regression in notebook display where <th> tags not used for :attr:`DataFrame.index` (:issue:`28204`).
 - Regression in :meth:`~DataFrame.to_csv` where writing a :class:`Series` or :class:`DataFrame` indexed by an :class:`IntervalIndex` would incorrectly raise a ``TypeError`` (:issue:`28210`)
--
+- Bug in :meth:`DataFrame.to_csv` where values were truncated when the length of ``na_rep`` was shorter than the text input data. (:issue:`25099`)
```

Comment:
Can you move this to v1.0.0 at this point?

Reply:
Sure, done. Moved to v1.0.0 under IO too. Removed the changes to v0.25.2, and edited last commit. Thanks @WillAyd